README.md

```diff
@@ -1,18 +1,48 @@
 gae_starter
 ===========
 
-This is a simple GAE starter showing off a few things:
+This is a simple GAE starter to show how easy it is to get a server running on App Engine, so you can focus on client-side aspects of your project.
 
-1. [bottle.py](http://bottlepy.org/) for handling requests
-2. [GAE](https://developers.google.com/appengine/) [cron](https://developers.google.com/appengine/docs/python/config/cron) and [task queues](https://developers.google.com/appengine/docs/python/taskqueue/)
-3. [Stripe](https://stripe.com/) for credit card payments
+If you need to just have a server return some hard-coded JSON and work on a real server response later, this is a good starter.
+If you need to save some simple data, you're better off using this than setting up a database and a server and all of that.  This is way easier.
+If you want a simple web application, this is a good starter as I've built over 10 sites using this basic framework.
 
-requirements
+It uses these basic technologies:
+
+1. [GAE](https://developers.google.com/appengine/)
+2. [bottle.py](http://bottlepy.org/) for handling requests, these are the @route annotations for assigning URL's to handler methods
+3. [jinja2](http://jinja.pocoo.org/docs/dev/) for HTML templating in static/template/ folder
+
+If you want to go further and add payments, cron jobs, or task queues, you may want to check out the older `full_example` branch.
+
+Setup for Windows
+============
+1. Install Python 2.7: https://www.python.org/downloads/ 
+2. Install App Engine SDK for Python https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi
```

Comment:
I know how to do links, but they're not all that useful when you want to copy paste directions into Slack or something...if the URL is visible here then it pastes visibly.